seriouslysean/monster-hunter-now-events
# Define a logging facade and use winston

## Issue

- Resolves #7 

## Testing Steps

1. Run any command to see the logging in action. Or you can run `npm run test:logger`

## Additional Information

1. You can add a transport layer on winston to handle rollbar though it.

(Sorry it took a while to send the PR, just go around to doing it this afternoon)
